test/sandbox/contracts/getAbi.ts

```diff
@@ -0,0 +1,23 @@
+// import Logger from 'helpers/Logger'
+
+import StablecoinConverter from '@gnosis.pm/dex-contracts/build/contracts/StablecoinConverter.json'
```

Comment:
Destructure here and everywhere else you don't need a whole json. Webpack and its ilk are smart enough to tree-shake other fields one level deep in json.